test/integration/statefulset/statefulset_test.go

```diff
@@ -149,7 +149,7 @@ func TestSpecReplicasChange(t *testing.T) {
 	})
 	savedGeneration := newSTS.Generation
 	if savedGeneration == oldGeneration {
-		t.Fatalf("failed to verify .Generation has incremented for sts %s", sts.Name)
+		t.Fatalf("failed to verify .Generation has incremented for StatefulSet %s", sts.Name)
```

Comment:
The similar lines (https://github.com/kubernetes/kubernetes/pull/114395/files#diff-002d19e38cf6a2efc053159b2a51434925fe14280e49e4a51188d3e60e841014R162 etc.) are not updated.
It is nice to have some consistent way to avoid multiple similar pull requests later.

Reply:
Updated. thanks